src/App.js

```diff
@@ -1,12 +1,24 @@
-import logo from './logo.svg';
+import { useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { fetchCountriesData } from './Redux/covidData/countriesCovidData';
+import HomePage from './components/home/Home';
+import Details from './components/details/Details';
 import './App.css';
 
-function App() {
+const App = () => {
```

Comment:
You have implemented some of the changes requested about this section well, but there are some of them that have not been done. Kindly follow the provided design template.

- [ ] The cells/ card of your list follow the alternating dual-color pattern. Each cell of the list item should have surrounding cells having different (alternating) background color(s). 
- [ ] The layout: composition and space between elements should be the same as the design template.


- [ OPTIONAL] I'll recommend using a font color with high contrast to the background color of the cards. As the main background image is blue and the cards are transparent, it clashes with the font color of the country names,  


_See images below_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png) | ![image](https://user-images.githubusercontent.com/49621279/164941716-e3ab795b-1893-4a6c-889b-247423d3cd32.png)